lib/core_refinements/string_refinement.rb

```diff
@@ -69,5 +69,20 @@ def titleize
     def surround(lhs, rhs = lhs)
       "#{lhs}#{self}#{rhs}"
     end
+
+    # @example
+    #
+    #   "on".normalize_keywords # => "\"age\""
+    #   "onMfa".normalize_keywords # => "\"onMFA\""
+    #   "onOauth".normalize_keywords # => "\"onOAuth\""
+    #
+    # @param [String] string
+    # @return [String]
+    def normalize_keywords
```

Comment:
Will depend on this in another generator, so making it a part of the `String` class refinement methods.